.gitignore

```
@@ -10,4 +10,5 @@ Manifest.toml
 benchmarks/benchmarks_output.json
 
 .ipynb_checkpoints
-*.ipynb
\ No newline at end of file
+.devcontainer/*
+*.ipynb
```

A minor comment, can you put `*.ipynb` together with `.ipynb_checkpoints` ?

```suggestion
*.ipynb

.devcontainer/*
```